<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WebCap</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>100</horstretch>
        <verstretch>100</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="documentLayout">
        <item>
         <widget class="QLabel" name="documentLabel">
          <property name="text">
           <string>PDF files</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="documentList"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="previewLayout">
        <item>
         <widget class="QLabel" name="previewLabel">
          <property name="text">
           <string>Preview</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="previewPicture"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>572</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionFileShowInExplorer"/>
    <addaction name="separator"/>
    <addaction name="actionFileSettings"/>
    <addaction name="separator"/>
    <addaction name="actionFileQuit"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelpAboutWebCap"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHilfe"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFileSelectFolder">
   <property name="text">
    <string>Select target folder...</string>
   </property>
  </action>
  <action name="actionFileShowInExplorer">
   <property name="text">
    <string>Show in Explorer</string>
   </property>
  </action>
  <action name="actionFileSettings">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionFileQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionHelpAboutWebCap">
   <property name="text">
    <string>About WebCap</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
